### PR TITLE
Replace EasyVCR submodule with NuGet package

### PR DESCRIPTION
**Pull request by nwithan8**
# Description
Now that EasyVCR is [officially on NuGet](https://www.nuget.org/packages/EasyVCR/) (yay!), let's stop using it as a submodule and instead import the NuGet package for our test suite.

- Drop-in replace Scotch/EasyVCR submodule with official NuGet package
- Update EasyVCR link in README.md
- Fix cassette path issue on AddressTest.cs

# Testing

Re-ran all tests with existing cassettes, re-recorded/replayed some cassettes, everything working as expected

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1 @@
-[submodule "Scotch"]
-	path = Scotch
-	url = https://github.com/EasyPost/scotch.git
+
```
Comment (jchen293) [inline .gitmodules:1]: Can this file be removed?

**Review by jchen293 — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (nwithan8) [reply .gitmodules:1]: You're right, I guess it could.

Comment (Justintime50) [reply .gitmodules:1]: Yes, this needs to be removed. You can follow this guide: https://gist.github.com/myusuf3/7f645819ded92bda6677

**Review by Justintime50 — CHANGES_REQUESTED** [review]
Will stamp once git submodule removed completely following the linked guide.

File: README.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ git submodule update --recursive
 
 ### Testing
 
-The test suite in this project was specifically built to produce consistent results on every run, regardless of when they run or who is running them. This project uses [EasyVCR](https://github.com/easypost/scotch) to record and replay HTTP requests and responses via "cassettes". When the suite is run, the HTTP requests and responses for each test function will be saved to a cassette if they do not exist already and replayed from this saved file if they do, which saves the need to make live API calls on every test run.
+The test suite in this project was specifically built to produce consistent results on every run, regardless of when they run or who is running them. This project uses [EasyVCR](https://www.nuget.org/packages/EasyVCR/) to record and replay HTTP requests and responses via "cassettes". When the suite is run, the HTTP requests and responses for each test function will be saved to a cassette if they do not exist already and replayed from this saved file if they do, which saves the need to make live API calls on every test run.
```
[the remark this answers is not in the document]
Comment (nwithan8) [reply README.md:127]: Well, the GitHub is linked via NuGet, versus not the other way around. The NuGet page has the README from GitHub as well.
At the end of the day, how do you find new packages in, say, Python? Do you search on PyPi, or do you find them via GitHub?

[the remark this answers is not in the document]
Comment (Justintime50) [reply README.md:127]: I personally search GitHub, but everyone is different. I guess we did update the README to read dynamically from GitHub so my initial concern is no longer a problem. Either way.

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**